package.json

```diff
@@ -24,11 +24,12 @@
         "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
     },
     "peerDependencies": {
-        "react": "^16.8.0 || ^17.0.0",
-        "react-dom": "^16.8.0 || ^17.0.0",
         "@emotion/react": "^11.4.1",
         "@emotion/styled": "^11.3.0",
-        "@material-ui/core": "^5.0.0-beta.4"
+        "@mui/material": "^5.0.0-rc.0",
+        "@mui/styles": "^5.0.0-rc.0",
```

Comment:
This branch no longer is dependant on `@mui/styles` afaik.
You added it here where it was no longer present.